.github/workflows/ci.yml

```diff
@@ -20,16 +20,16 @@ jobs:
     steps:
 
     - name: Cancel Previous Runs
-      uses: styfle/cancel-workflow-action@bb6001c4ea612bf59c3abfc4756fbceee4f870c7 # 0.10.0
+      uses: styfle/cancel-workflow-action@0.10.0 # 0.10.0
       with:
         access_token: ${{ github.token }}
 
     - name: Install Protoc
-      uses: arduino/setup-protoc@v1
+      uses: arduino/setup-protoc@31a047f6ff1b725b36c561770f81012e2f165926
 
     - uses: actions/checkout@v3
 
-    - uses: Swatinem/rust-cache@6720f05bc48b77f96918929a9019fb2203ff71f8 # v2.0.0
+    - uses: Swatinem/rust-cache@76686c56f2b581d1bb5bda44b51f7e24bd9b8b8e
```

Comment:
As far as I can see, this hash is not yet part of a released version. I'd prefer if we just not touch this line at all and revert back to exactly what we had :)